components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java

```diff
@@ -317,4 +319,37 @@ public void updateRoleName(int tenantId, String oldRoleName, String newRoleName)
         }
     }
 
+    public String[] getGroupNameList(String attributeName, String searchAttribute, Integer tenantId)
```

Comment:
Missing java doc comment. All the public methods should be having a java doc comment